batch/create/create_batch_labels_runnable.js

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+async function main() {
+  // [START batch_labels_runnable]
+  // Imports the Batch library
+  const batchLib = require('@google-cloud/batch');
+  const batch = batchLib.protos.google.cloud.batch.v1;
+
+  // Instantiates a client
+  const batchClient = new batchLib.v1.BatchServiceClient();
+
+  /**
+   * TODO(developer): Update these variables before running the sample.
+   */
+  // Project ID or project number of the Google Cloud project you want to use.
+  const projectId = await batchClient.getProjectId();
+  // Name of the region you want to use to run the job. Regions that are
+  // available for Batch are listed on: https://cloud.google.com/batch/docs/get-started#locations
+  const region = 'europe-central2';
+  // The name of the job that will be created.
+  // It needs to be unique for each project and region pair.
+  const jobName = 'batch-labels-runnable';
+  // Name of the label1 to be applied for your Job.
+  const labelName1 = 'runnable_label_name_1';
```

Comment:
Hi, quick question. Is it possible to use the same variables as we have in here https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/batch/snippets/src/main/java/com/example/batch/CreateBatchRunnableLabel.java#L17-L142 ?

Reply:
names adjusted